### ISSUE TITLE
a11y issues with links and images on About Us and Contributor pages

### ISSUE THREAD

**Issue by StephNathai** [anitab-org/anitab-org.github.io#379]
### Describe the bug
- links on the About Us and Contributor pages are not a11y compliant due to: _elements must meet minimum contrast ratio thresholds_
- member image links on Contributor page are not a11y compliant due to: _links must have discernible text_

### To Reproduce
1. Go to [https://anitab-org.github.io/](https://anitab-org.github.io/) and run a11y audit on the About Us tab and Contributor tab to see the issues called out above in the audit

### Expected behavior
- there should be no a11y issues on these page

**Comment by StephNathai** [comment]
I will put up a PR shortly to resolve this issue
